web/includes/download_functions.php

```diff
@@ -18,6 +18,10 @@
 // Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 // 
 
+function cmpDataTime($a, $b) {
+    return strcmp(strtotime($a->StartDateTime), strtotime($b->StartDateTime));
```

Comment:
strtotime returns an integer so we shouldn't really use strcmp... can just use <=> I think